README.arm.md

```diff
@@ -125,47 +132,32 @@ These tutorials will end up installing Ubuntu 12.04, and you have to
 upgrade to Ubuntu 14.04, or install Ubuntu 14.04 from scratch by
 finding appropriate `crouton` help.
 
-## Scaleway cloud hosted ARM servers
+#### Scaleway cloud hosted ARM servers
 
 On the current [Scaleway](http://scaleway.com) ARM servers, the Julia
 build works out of the box.
 
-## nVidia Jetson TX2
+## AArch64 (ARMv8)
 
-Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html) platform with minimal configuration changes. A full multi-threaded build, including LLVM, will complete in around two hours. All tests pass and CUDA functionality is available through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
+### Device specific instructions
 
-Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
+#### nVidia Jetson TX2
 
-```
-sudo apt-get install libssl-dev
-```
+Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html)
+platform with minimal configuration changes. A full multi-threaded build, including LLVM,
+will complete in around two hours. All tests pass and CUDA functionality is available
+through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
 
-### Julia 0.5.1
-
-The easiest method to build Julia 0.5.1 is to use system provided versions of BLAS and LAPACK:
-
-```
-sudo apt-get install libopenblas-dev liblapack-dev
-```
-
-Configure Make.user as follows:
+Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
 
 ```
-MARCH=armv8-a
-JULIA_CPU_TARGET=cortex-a57
-override USE_SYSTEM_BLAS=1
-override USE_SYSTEM_LAPACK=1
+sudo apt-get install libssl-dev
```

Comment:
At the moment the 'general' build instructions are part of the 32-bit specific section, rather than  the general content, since I haven't yet had to do anything other than install some standard libraries and run `make` on the `AArch64` target to which I have access.

I could move the 'general' instructions into the non-architecture-specific material, but note that at present these:
1. make no distinction between requisite build software and system libraries which can be installed in order to avoid building dependencies
2. refer to specific `gcc` versions that might not be appropriate (for example, I am using 5.4 on `AArch64`)
3. don't specify `libssl` (is it just a peculiarity of the nVidia customised distribution that this wasn't installed by default?).

I can remove the general build software from the current list of requirements (and simply refer to the main Julia README), at which point this is fairly generic advice. But I don't have any AArch32 systems on which to test modified instructions, hence my hesitance to touch this section.